tests/partitioninfo.spec.coffee

```diff
@@ -69,3 +70,59 @@ describe 'Partitioninfo:', ->
 					type: 131
 				}
 			])
+
+	describe '.get() disk 2', ->
+
+		it 'should be able to get the first logical partition of the extended one', ->
+			promise = partitioninfo.get(DISK2_PATH, 5)
+			m.chai.expect(promise).to.eventually.become
+				offset: 3145728
+				size: 1048576
+				type: 131
+
+		it 'should be able to get the second logical partition of the extended one', ->
+			promise = partitioninfo.get(DISK2_PATH, 6)
+			m.chai.expect(promise).to.eventually.become
+				offset: 5242880
+				size: 1048576
+				type: 131
+
+		it 'should return an error when asked for a non existing partition', ->
+			promise = partitioninfo.get(DISK2_PATH, 10)
+			m.chai.expect(promise).to.be.rejectedWith('Partition not found: 10.')
+
+	describe '.getPartitions() disk 2', ->
+		it 'should list all partitions of a disk image', ->
+			promise = partitioninfo.getPartitions(DISK2_PATH)
+			m.chai.expect(promise).to.eventually.become([
+				{
+					'offset': 1048576
+					'size': 1048576
+					'type': 131
+				}
+				{
+					'offset': 2097152
+					'size': 7340032
+					'type': 5
+				}
+				{
+					'offset': 9437184
+					'size': 1048576
+					'type': 131
+				}
+				{
+					'offset': 3145728
+					'size': 1048576
+					'type': 131
+				}
+				{
+					'offset': 5242880
+					'size': 1048576
+					'type': 131
+				}
+				{
+					'offset': 7340032
+					'size': 1048576
+					'type': 131
+				}
```

Comment:
I'll add a note in the README about the partitions order.
`ignoreExtended` is a good idea, I'll add that.